Dockerfile

```diff
@@ -1,38 +1,117 @@
 ARG VERSION=unspecified
+ARG PY_VERSION=3.10.0
 
-FROM python:3.10.0-alpine
+FROM python:${PY_VERSION}-slim-bullseye AS compile-stage
 
 ARG VERSION
 
 # For a list of pre-defined annotation keys and value types see:
 # https://github.com/opencontainers/image-spec/blob/master/annotations.md
 # Note: Additional labels are added by the build workflow.
-LABEL org.opencontainers.image.authors="mark.feldhousen@cisa.dhs.gov"
+LABEL org.opencontainers.image.authors="jeremy.frasier@cisa.dhs.gov"
```

Comment:
Do we care about the `LABEL`s anywhere except in the final image?

Reply:
I don't know...I figured it wouldn't hurt anything to have them there.  I'm happy to remove them if the group thinks they should not be there.

Reply:
I'd say ditch em.  Can't have a bug in a line of code that doesn't exist.  
Plus I could totally see me updating this in the future and not understanding why the final image tags don't update.

Reply:
Enjoy commit 8f2554a!

Reply:
So hot!  🌶️